jnigen/test/kotlin_test/generated_files_test.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:jnigen/jnigen.dart';
+import 'package:test/test.dart';
+import 'package:path/path.dart' hide equals;
+
+import 'generate.dart';
+import '../test_util/test_util.dart';
+
+void main() async {
+  test("Generate and compare bindings for kotlin_test", () async {
+    await generateAndCompareBindings(
+      getConfig(),
+      join(testRoot, "lib", "kotlin.dart"),
+      join(testRoot, "src"),
+    );
+  }); // test if generated file == expected file
+  test("Generate and analyze bindings for kotlin_test - pure dart", () async {
+    await generateAndAnalyzeBindings(
+      getConfig(BindingsType.dartOnly),
+    );
+  }); // test if generated file == expected file
+}
```

Comment:
Should the generated bindings also be executed?

That we we can test whether the code actually works instead of only passing Dart static analysis. :)

Reply:
That logic resides in `bindings_test.dart`

Reply:
Other tests are also similar, I created an issue for all of them: https://github.com/dart-lang/jnigen/issues/183